splunk-otel-android/src/main/java/com/splunk/rum/RumInitializer.java

```diff
@@ -143,7 +144,13 @@ SpanExporter buildExporter(ConnectionUtil connectionUtil) {
             // we'll do our best to hang on to the spans with the wrapping BufferingExporter.
             ZipkinSpanExporter.baseLogger.setLevel(Level.SEVERE);
         }
-        return new BufferingExporter(connectionUtil, ZipkinSpanExporter.builder().setEndpoint(endpoint).build());
+        ZipkinSpanExporter zipkinSpanExporter = ZipkinSpanExporter.builder().setEndpoint(endpoint).build();
+        SpanExporter throttlingExporter = ThrottlingExporter.newBuilder(zipkinSpanExporter)
+                .categorizeByAttribute(SplunkRum.COMPONENT_KEY)
+                .maxSpansInWindow(100)
+                .windowSize(Duration.ofSeconds(30))
```

Comment:
@jkwatson should I add these to `Config`? I feel kind of bad hard-coding them here...

Reply:
@johnbley is this something iOS has made configurable?

Reply:
Not currently; I don't expect it would be necessary.

Reply:
let's leave them out for now. If someone needs it, we can always add it later.